src/drivers/mct/cime_config/config_compsets.xml

```diff
@@ -46,6 +46,26 @@
   </compset>
 
   <compset>
+    <alias>ADSOM</alias>
+    <lname>2000_DATM%NYF_SLND_DICE%SSMI_DOCN%SOM_DROF%NYF_SGLC_SWAV_TEST</lname>
+  </compset>
+
+  <compset grid="a%1.9x2.5|a%0.9x1.25">
```

Comment:
Does this grid attribute actually accomplish anything? I tried creating test `SMS.f10_f10.ADSOMAQP.roo2_gnu` and it got through initial case creation just fine, making me think that the grid attribute wasn't doing anything. (I had been expecting the grid attribute to stop me from creating a case with this compset unless I'm using one of the above grids, but that doesn't seem to be the case.)